src/main/java/com/jfrog/ide/common/deptree/DepTree.java

```diff
@@ -0,0 +1,26 @@
+package com.jfrog.ide.common.deptree;
+
+import java.util.Map;
+
+public class DepTree {
```

Comment:
It is a bit confusing that we have both DepTree and DependencyTree classes...
What do you think?